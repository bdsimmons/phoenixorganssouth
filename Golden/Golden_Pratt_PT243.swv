<?xml version="1.0" encoding="UTF-8"?>
<SWiSHvideo version="2.0" fileVersion="1.0" build="2005.12.04"
 originalFolder="C:\Documents and Settings\David\My Documents\My Videos\FlashMovies\Golden"
>
	<video
		file="..\..\Golden_Pratt_PT243.wmv"
		bitrate="192000"
		quality="80"
		keyInterval="10"
		deinterlace="0"
		deepmotiondetection="0"
		outputFps="20.0"
		xResize="100%"
		yResize="100%"
		crop="0,0,0,0"
	/>
	<audio
		alt=""
		outputFlag="1"
		fadeOutTime="0"
		frequency="22050"
		mono="0"
		compression="1"
		param="64 (LAME MP3)"
		saveTo=""
	/>
	<output
		file=".\Golden_Pratt_PT243.swf"
		format="0"
		borderWidth="0"
		bgColor="0"
		stop="3"
		url=""
		target=""
		html="1"
		events="0"
		controls="Ultra Minimalist"
	/>
	<variables>
		<var name="umTrackColor" value="FFFFFF" type="2"/>
		<var name="umSlideColor" value="CCCCCC" type="2"/>
		<var name="umShadeAlpa" value="60" type="2"/>
		<var name="umRollColor" value="E8E8E8" type="2"/>
		<var name="umPressColor" value="D4D4D4" type="2"/>
		<var name="umLoadColor" value="DDDDDD" type="2"/>
		<var name="umInfoBar" value="on" type="2"/>
		<var name="umIconColor" value="000000" type="2"/>
		<var name="umButtonColor" value="FFFFFF" type="2"/>
		<var name="umBorderColor" value="000000" type="2"/>
		<var name="svChapterAlpha" value="70" type="2"/>
		<var name="svChapterMenu" value="off" type="2"/>
		<var name="svFlashVersion" value="6" type="2"/>
		<var name="svDetectFlash" value="on" type="2"/>
		<var name="svBufferTime" value="5" type="2"/>
		<var name="svBuffer" value="on" type="2"/>
		<var name="svBufferText" value="Buffering ... " type="2"/>
		<var name="svSetupText" value="Initializing Video" type="2"/>
		<var name="svTitle" value="Golden Pratt PT243" type="1"/>
	</variables>
	<overlay
		file=""
		active="0"
		valign="2"
		halign="0"
	/>
	<events>
	</events>
</SWiSHvideo>
