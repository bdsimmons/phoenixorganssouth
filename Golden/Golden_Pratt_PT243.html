<html>
<body bgcolor=#000000>
<center>
<object classid="clsid:D27CDB6E-AE6D-11cf-96B8-444553540000" width="320" height="252">
  <param name="MOVIE" value="Golden_Pratt_PT243.swf">
  <param name="QUALITY" value="HIGH">
  <param name="BGCOLOR" value="#000000">
     <embed src="Golden_Pratt_PT243.swf"
            quality="high"
            type="application/x-shockwave-flash"
            bgcolor="#000000"
            width="320" height="252">
</object></center>
</body>
</html>
